File: starter-bots/KotlinBot/src/main/kotlin/arctica/service/model/PopulationTier.kt
package arctica.service.model

data class PopulationTier(
    val level: Int,
    val name: String,
    val maxPopulation: Int,
    val populationChangeFactorRange: List<Double>,
    val tierResourceConstraints: TierResourceConstraints
)

data class TierResourceConstraints(
    val Food: Int,
    val Wood: Int,
    val Stone: Int
)

File: starter-bots/PythonBot/PopulationTier.py
class PopulationTier:
	def __init__(self, level, name, max_population, population_change_factor_range, tier_resource_constraints) -> None:
		self.level = level
		self.name = name
		self.maxPopulation = max_population
		self.populationChangeFactorRange = population_change_factor_range
		self.tierResourceConstraints = tier_resource_constraints